rules/autoconsent/canva.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "canva.com",
+  "prehideSelectors": ["div[role=\"dialog\"] a[data-anchor-id=\"cookie-policy\"]"],
+  "detectCmp": [{ "exists": "div[role=\"dialog\"] a[data-anchor-id=\"cookie-policy\"]" }],
+  "detectPopup": [{ "exists": "div[role=\"dialog\"] a[data-anchor-id=\"cookie-policy\"]" }],
+  "optIn": [
+    { "click": "div[role=\"dialog\"] button:nth-child(1)" }
+  ],
+  "optOut": [
+    {
+      "click": "div[role=\"dialog\"] button:nth-child(2)"
```

Comment:
For me, just this step is sufficient (clicks on 'Reject all cookies'). The rest of the steps then fail because the popup is gone.

Reply:
From California there's no "reject all" button so I just added a condition to match California and Europe's requirements.